tenzir/integration/README.md

```diff
@@ -8,7 +8,7 @@ the test output against a known reference file.
 
 # Running integration tests
 
-To run all tests, execute the `integration` target from cmake.
+To run all tests, execute the `integration` target from CMake.
```

Comment:
Would it be target inflation if we had one target per test file? If not, I would love to switch away from manually invoking bats entirely. But I often only care for a single suite.

Reply:
I think having one target per test suite is an excellent idea. I've written down a note to do that some time in the future.